pjp.tm11/src/main/webapp/login/css/util.css

```diff
@@ -0,0 +1,2993 @@
+/*[ FONT SIZE ]
+///////////////////////////////////////////////////////////
+*/
+.fs-1 {font-size: 1px;}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5662373)